scripts/linters/pylint_extensions.py

```diff
@@ -2109,6 +2109,45 @@ def visit_call(self, node):
                     break
 
 
+class DisallowBlankLinesBelowFunctionDefinitionChecker(checkers.BaseChecker):
+    """Custom pylint checker to ensure there are no blank lines
+    below function definition.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+    name = 'disallow-blank-lines-below-function-definition'
+    priority = -1
+    msgs = {
+        'C0034': (
+            'Please remove the blank line(s) below the function definition.',
+            'no-blank-lines-below-function-definiton.',
+            'Disallow blank lines below function definition.'
+        ),
+    }
+
+    def visit_functiondef(self, node):
+        """Visits every function definition and ensures there are no blank lines
+        below function definition.
+
+        Args:
+            node: astroid.scoped_nodes.FunctionDef. Node for a function
+                or method definition in the AST.
+        """
+
+        line_number = node.fromlineno
+
+        for i in python_utils.RANGE(999):
+            line = linecache.getline(node.root().file, line_number + i).strip()
+            if ':' in line:
+                line_number += i
+                break
+        line_after_function_def = linecache.getline(
+            node.root().file, line_number + 1).strip()
```

Comment:
Are you sure this is working fine? Because we are expecting docstring after the `:`, right?)

Reply:
@DubeySandeep I manually added space below function definition in a file and ran the linter, and it showed me the error. 

And I have not assumed that every function has a docstring, @sajalasati also guided me on this. Like see any function in [this](https://github.com/oppia/oppia/blob/develop/scripts/linters/pylint_extensions_test.py) , no ( or very less)  functions have docstring.  
My approach is simply finding a colon in a file, as it is a point where function definiton ends, and check if the next line is empty or not. It is not dependent on docstring.

Reply:
@Mayank-gaur I had a discussion with the linter team and it looks like we are having a misunderstanding with the requirement of this check. To clarify, we want to avoid a new line after function definition
CORRECT:
```
def some_function():
"""Some docstring."""

Reply:
Sorry for the misunderstanding, let me know if you are able to understand the requirement now or if you have any questions?

Reply:
Okay Thanks @DubeySandeep .I have understood the requirements. I will update this PR with a lint ensuring space below function docstring.